frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/validation/AuthAppCodeProcessor.java

```diff
@@ -84,6 +93,21 @@ public Optional<ErrorResponse> validateCode(CodeRequest codeRequest) {
         return Optional.empty();
     }
 
+    @Override
+    public void processSuccessfulCodeRequest(String ipAddress, String persistentSessionId) {
+        if (codeRequest.getJourneyType().equals(JourneyType.REGISTRATION)) {
```

Comment:
This will be eventually turned into a switch statement. AccountRecovery journey type will remove the old method and add the new one whilst SignIn journey type will remove the block if one is present